tests/conftest.py

```diff
@@ -144,3 +144,8 @@ def fixture_german(app_context):
 @pytest.fixture(name="hindi")
 def fixture_hindi(app_context):
     return _get_test_language("Hindi")
+
+
+@pytest.fixture(name="mandarin_chinese")
```

Comment:
This has changed a bit in `develop` since this PR was created -- this should now be

```
@pytest.fixture(name="mandarin_chinese")
def fixture_generic(app_context):
    return _get_test_language("Mandarin Chinese")   # I'm assuming this is correct ...
```

actually this change will also need a change in the language_defs submodule, which gets complicated/annoying.  We can work through that.